app/models/support_ticket.rb

```diff
@@ -0,0 +1,41 @@
+class SupportTicket
+  def initialize(params:)
```

Comment:
is it worth including ActiveModel here, so it can do validations?

Reply:
@aldavidson I didn't do this because @aliuk2012 had already created form objects for each stage that included `ActiveModel::Model`. They do validations as the user progresses through each of the forms.